lib/iris/cube.py

```diff
@@ -1919,6 +1957,73 @@ def coord_system(self, spec=None):
 
         return result
 
+    def _a_meshcoord(self):
+        mesh_coords = self.coords(mesh_coords=True)
+        if mesh_coords:
+            result = mesh_coords[0]
+        else:
+            result = None
+        return result
+
+    def mesh(self):
```

Comment:
@pp-mo We're never (never say never) going to pass parameters to this, so could we err on the side of starting this methods life as a read-only property i.e.,
```python
@property
def mesh(self):
    ...
```
This makes the usage a lot cleaner IMHO i.e.,
```python
cube.mesh
```
rather than
```python
cube.mesh()
```
I realise that `Cube.coord_system` is indeed a method, but that's a slightly different beast... plus it does accept `kwargs`.